examples/plot_decoding_tutorial.py

```diff
@@ -167,40 +167,39 @@
 # fmri data to restrict the classification to the face vs cat discrimination.
 # As a consequence, the input data is much small (i.e. fmri signal is shorter):
 condition_mask = conditions.isin(['face', 'cat'])
-fmri_masked = fmri_masked[condition_mask]
-print(fmri_masked.shape)
+
+###########################################################################
+# Because the data is in one single large 4D image, we need to use
+# index_img to do the split easily.
+from nilearn.image import index_img
+fmri_niimgs = index_img(fmri_filename, condition_mask)
 
 ###########################################################################
 # We apply the same mask to the targets
 conditions = conditions[condition_mask]
+# Convert to numpy array
+conditions = conditions.values
 print(conditions.shape)
 
-
 ###########################################################################
-# Decoding with an SVM
-# ---------------------
-#
-# We will now use the `scikit-learn <http://www.scikit-learn.org>`_
-# machine-learning toolbox on the fmri_masked data.
+# Decoding with Support Vector Machine 
+# --------------------------------------
 #
-# As a decoder, we use a Support Vector Classification, with a linear
-# kernel.
-#
-# We first create it:
-from sklearn.svm import SVC
-svc = SVC(kernel='linear')
-print(svc)
+# As a decoder, we use a Support Vector Classification, with a linear kernel. We
+# first create it using by using :class:`nilearn.decoding.Decoder`.
+from nilearn.decoding import Decoder
+decoder = Decoder(estimator='svc', mask=mask_filename, standardize=True)
```

Comment:
I believe that we no longer need to use the NiftiMasker in this example. Can you remove the corresponding lines / section in the above?